src/E1_coldflow/config.h

```diff
@@ -11,9 +11,10 @@
 std::string str_file_name = "E1_speed_test_results.txt";
 const char * file_name = str_file_name.c_str();
 
-const int numCryoTherms = 2;
-int cryoThermAddrs[numCryoTherms] = {0x60, 0x67};
-_themotype cryoTypes[numCryoTherms] = {MCP9600_TYPE_J, MCP9600_TYPE_T};
+const int numCryoTherms = 4;
+// therm[2] = lox adapter tree pt, therm[3] = lox adapter tree gems
+int cryoThermAddrs[numCryoTherms] = {0x60, 0x67, 0x62, 0x64};
+_themotype cryoTypes[numCryoTherms] = {MCP9600_TYPE_J, MCP9600_TYPE_T, MCP9600_TYPE_T, MCP9600_TYPE_K};
```

Comment:
You guys will need to set up and verify these this weekend. 
Addresses defined w/ voltage inputs on a specific pin.
Which type thermocouple each is. 
Make sure that the thermocouple on lox adapter tree pt is at index 2 and that on lox adapter tree gems is at index 3